src/main/java/com/google/longrunning/OperationsSettings.java

```diff
@@ -66,6 +66,9 @@
   /** The default port of the service. */
   private static final int DEFAULT_SERVICE_PORT = 443;
 
+  private static final String DEFAULT_GENERATOR_NAME = "gapic";
+  private static final String DEFAULT_GENERATOR_VERSION = "0.0.5";
```

Comment:
@garrettjonesgoogle @shinfan I was wondering why you always need the `gccl` version, even when the client is using GAPIC directly. Then I just answered my own question with my preceding response. This is unfortunate, because it means we can never distinguish between usage of the manual and GAPIC layers, except possibly on Python. We want to be able to do this, so I need to think about this a bit more.